crates/turbopack-ecmascript/tests/tree-shaker/analyzer/shared-and-side-effects/output.md

```diff
@@ -0,0 +1,461 @@
+# Items
+
+Count: 12
+
+## Item 1: Stmt 0, `Normal`
+
+```js
+console.log("Hello");
+
+```
+
+- Side effects
+- Reads: `console`
+
+## Item 2: Stmt 1, `VarDeclarator(0)`
+
+```js
+const value = externalFunction();
+
+```
+
+- Declares: `value`
+- Reads: `externalFunction`
+- Write: `value`
+
+## Item 3: Stmt 2, `VarDeclarator(0)`
+
+```js
+const value2 = externalObject.propertyWithGetter;
+
+```
+
+- Declares: `value2`
+- Reads: `externalObject`
+- Write: `value2`
+
+## Item 4: Stmt 3, `Normal`
+
+```js
+externalObject.propertyWithSetter = 42;
+
+```
+
+- Side effects
+- Reads: `externalObject`
+
+## Item 5: Stmt 4, `VarDeclarator(0)`
+
+```js
+const value3 = externalFunction();
+
+```
+
+- Declares: `value3`
+- Reads: `externalFunction`
+- Write: `value3`
```

Comment:
This should not be flagged as side effect as the function call was annotated with PURE.